.github/workflows/ci.yml

```diff
@@ -18,8 +18,11 @@ jobs:
         run: |
           TAG=$(echo $GITHUB_SHA | head -c7)
           IMAGE="docker.pkg.github.com/toggle-corp/kitab-server"
-          echo ::set-output name=tagged_image::${IMAGE}:${TAG}
-          echo ::set-output name=tag::${TAG}
+          #echo ::set-output name=tagged_image::${IMAGE}:${TAG}
+          #echo ::set-output name=tag::${TAG}
+          echo "{tagged_image}=${IMAGE}:${TAG}" >> $GITHUB_OUTPUT
+          echo "{tag}=${TAG}" >> $GITHUB_OUTPUT
```

Comment:
For research purposes, can you please share the documentation which shows this implementation?

Reply:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/